execution_engine/src/core/engine_state/mod.rs

```diff
@@ -785,6 +785,40 @@ where
         Ok(BalanceResult::Success { motes, proof })
     }
 
+    /// Get the total supply for an era targeted by the state hash.
```

Comment:
I wouldn't say "for an era" here. Execution engine is ignorant of eras. It just deals with the global state as it is at some particular hash. So, maybe something like:

> Get the total supply of tokens at the given state hash.

Reply:
bart is correct